crawler/helpers.py

```diff
@@ -485,6 +494,30 @@ def __init__(self):
         self.max_errors = 5
         self.short_display_description = "Result not aligned with CHn-Results"
 
+class AggregateType22(AggregateTypeBase):
```

Comment:
This might need a comment to explain what it is.

Reply:
Oh, okay,  github context wasn't making it clear that there are tonnes of these.

Reply:
Yeah this is the standard way in which known errors are added, to be referred to by TYPE in the calling code